SPECS/qemu/CVE-2024-4693_crash_fix.patch

```diff
@@ -0,0 +1,90 @@
+From 352e1d101fa3cfb161ee951af754bff791ff2241 Mon Sep 17 00:00:00 2001
+From: Cindy Lu <lulu@redhat.com>
+Date: Wed, 22 May 2024 13:10:24 +0800
+Subject: [PATCH] virtio-pci: Fix the use of an uninitialized irqfd.
```

Comment:
Can I ask you to merge this with the `CVE-2024-4693.patch` file? No need to split this into two separate ones, since they both are meant to fix one CVE.